File: java-source/src/test/java/net/corda/accord/contract/PromissoryNoteSettleTests.java
package net.corda.accord.contract;

import net.corda.core.contracts.Amount;
import net.corda.core.contracts.CommandData;
import net.corda.core.contracts.PartyAndReference;
import net.corda.core.contracts.TypeOnlyCommandData;
import net.corda.core.identity.AbstractParty;
import net.corda.core.utilities.OpaqueBytes;
import net.corda.finance.contracts.asset.Cash;
import net.corda.testing.node.MockServices;

import java.util.Arrays;
import java.util.Currency;

import static net.corda.testing.node.NodeTestUtils.ledger;

/**
 * TODO: Modify tests to adapt from IOU to promissory note.
 */

public class PromissoryNoteSettleTests {

    public interface Commands extends CommandData {
	    class DummyCommand extends TypeOnlyCommandData implements Commands{}
    }

    static private final MockServices ledgerServices = new MockServices(Arrays.asList("net.corda.accord"));


}

File: kotlin-source/src/test/kotlin/net/corda/accord/contract/PromissoryNoteSettleTests.kt
package net.corda.accord.contract

import net.corda.core.contracts.TypeOnlyCommandData
import net.corda.testing.node.MockServices


class PromissoryNoteSettleTests {
    // A pre-defined dummy command.
    class DummyCommand : TypeOnlyCommandData()
    var ledgerServices = MockServices(listOf("net.corda.accord"))

}

